src/Symfony/Component/Form/FormConfigBuilder.php

```diff
@@ -761,6 +770,16 @@ public function getFormConfig()
         return $config;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function setIsEmptyCallback(?callable $isEmptyCallback)
```

Comment:
`: self`?

Reply:
Return type of the method is `$this` (like all the other interface methods). We will add `: static` later and if the RFC makes it.